현정빈/RestApiStudy/src/main/java/com/example/restapistudy/dto/MovieDto.java

```diff
@@ -0,0 +1,37 @@
+package com.example.restapistudy.dto;
+
+import com.example.restapistudy.domain.Movie;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+public class MovieDto
+{
+    private Long id; //아이디
+    private String title; //영화제목
+    private String director; //감독
+    private int releaseYear; //출시일
+    private double rating; //평점
+
+    @Builder
+    public MovieDto(Long id, String title, String director,int releaseYear, double rating)
+    {
+        this.id = id;
+        this.title = title;
+        this.director = director;
+        this.releaseYear = releaseYear;
+        this.rating = rating;
+    }
```

Comment:
중괄호는 하나의 라인으로 작성되는게 관례에요!
```suggestion
    @Builder
    public MovieDto(Long id, String title, String director,int releaseYear, double rating) {
        this.id = id;
        this.title = title;
        this.director = director;
        this.releaseYear = releaseYear;
        this.rating = rating;
    }
```

Reply:
제가 착각하고 중괄호 위치를 바꿨네요.. 다음 과제부터 지키도록 하겠습니다!